stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/viewholder/OnlyMediaAttachmentsViewHolder.kt

```diff
@@ -0,0 +1,41 @@
+package io.getstream.chat.android.ui.messages.adapter.viewholder
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.core.view.isVisible
+import com.getstream.sdk.chat.ImageLoader.load
+import com.getstream.sdk.chat.adapter.MessageListItem
+import io.getstream.chat.android.client.models.Attachment
+import io.getstream.chat.android.ui.R
+import io.getstream.chat.android.ui.databinding.StreamItemMessageAttachmentsOnlyBinding
+import io.getstream.chat.android.ui.messages.adapter.BaseMessageItemViewHolder
+
+public class OnlyMediaAttachmentsViewHolder(
+    parent: ViewGroup,
+    internal val binding: StreamItemMessageAttachmentsOnlyBinding = StreamItemMessageAttachmentsOnlyBinding.inflate(
+        LayoutInflater.from(
+            parent.context
+        ),
+        parent,
+        false
+    )
+) : BaseMessageItemViewHolder<MessageListItem.MessageItem>(binding.root) {
+
+    override fun bindData(data: MessageListItem.MessageItem) {
+        if (data.message.attachments.firstOrNull()?.type == "image") {
+            showImage(data.message.attachments.first(), data.isMine)
+        }
```

Comment:
What happens in the other case, on the `else` branch? Can that ever happen?

Reply:
It's just first part of Attachments Media VH, I worked out only this case